src/Application.php

```diff
@@ -141,7 +146,7 @@ protected function initSabreDAV() {
                 $this['config']['auth']['type'],
                 $this['config']['auth']['realm'],
                 $this['pdo'],
-                null
+                '/dav.php'
```

Comment:
Might have to remove this again. This is only correct for users that run baikal on the root of their server.